datacenter/dtr/2.2/guides/admin/configure/external-storage/s3.md

```diff
@@ -0,0 +1,116 @@
+---
+description: S3 storage configuration for Docker Trusted Registry
+keywords: docker, dtr, storage driver, s3 storage, s3 compatible
+title: Configuring S3 storage within DTR
```

Comment:
Our titles use the regular form, not gerund. We also don't need to provide all the context in the title, so we can change it just to

Amazon S3

Here's a preview of what I want us to achieve
<img width="393" alt="screen shot 2017-02-21 at 17 18 02" src="https://cloud.githubusercontent.com/assets/1525937/23192706/c8e67fe6-f859-11e6-8489-138ede0e2ebb.png">